hackathon_site/hackathon_site/urls.py

```diff
@@ -15,8 +15,41 @@
 """
 from django.contrib import admin
 from django.urls import path, include
+from django.conf.urls import url
+
+from rest_framework import permissions
+from drf_yasg.views import get_schema_view
+from drf_yasg import openapi
+
+schema_view = get_schema_view(
+    openapi.Info(
+        title="Snippets API",
+        default_version="v1",
+        description="Test description",
+        terms_of_service="https://www.google.com/policies/terms/",
+        contact=openapi.Contact(email="contact@snippets.local"),
+        license=openapi.License(name="BSD License"),
+    ),
+    public=True,
+    permission_classes=(permissions.AllowAny,),
+    #    url="/api/",
+)
+
 
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("api/", include("api.urls", namespace="api")),
+    url(
```

Comment:
We don't really need the json/yaml view of the schema, so imo I would leave it out. The Swagger UI is just there at the moment to provide convienent documentation for us.